### PR TITLE
Console.log wrapper & data file changes & begining of backend testing

### PR DESCRIPTION
**Pull request by Michiocre**
Wrapped the console.log command and added a api function to access older logs (currently open to anyone).

Also small update to the current data storage system

### CODE REVIEW

File: backend/api/callback.js [hunk 1]
```diff
@@ -41,22 +40,33 @@ module.exports = async (req, res) => {
     } catch (e) {
         fetchData = null;
     }
+
+
```
Comment (Gabsii) [inline backend/api/callback.js]: remove empty line

File: backend/handler/handler.js [hunk 2]
```diff
@@ -82,9 +87,38 @@ const allowCors = fn => async (req, res) => {
...
+        // eslint-disable-next-line no-console
+        console.error(message);
+    } else {
+        // eslint-disable-next-line no-console
+        console.log(message);
+    }
+
+    let datetime = new Date();
+
+    fs.appendFile(logsLoc, `[${datetime.toISOString()}] ${type ? type : 'log'} : ${message} <br>`, function (err) {
```
Comment (Gabsii) [inline backend/handler/handler.js]: the ternary can be removed if you set `type = 'log'` as a default parameter

**Review by Gabsii — CHANGES_REQUESTED** [review]

**Comment by Gabsii** [comment]
Is this PR ready for review? Otherwise, turn it into a draft PR please.

**Comment by Michiocre** [comment]
Is ready for review now.
The backend testing hasn't really been implemented yet only the framework.
Adding new test will be pretty easy from now on.
Im not too sure about the testing structure so it might get changed again.

I wont be able to split the data files into one for each room since i have a few functions like getRoomByName, getRoomByHostToken, getRoomById. And for these to work i need to have all the room in a array.
So by keeping all the rooms in a single file there is less time spent on file management.
The files have been optimized a bit there shouldn't be as much unused data floating around.

I also finally fixed the labeler correctly so you shouldn't be getting a failed workflow email with every push.

**Pull request closed without merging**